docs/orgs.md

```diff
@@ -70,6 +70,29 @@ This will create the organization data in a file `group-<snyk_group_id>-bitbucke
 This will create the organization data in a file `group-<snyk_group_id>-bitbucket-cloud-orgs.json`
 
 
+## Azure
+**Please note that for Azure, this step needs to be done manually**
+Since Azure has no API call for getting the Azure Organizations, the Orgs file must be created manually for the next commands to run:
```

Comment:
can we link to something here like their docs to confirm this claim?

Reply:
Done